internal/controller/supervisorconfig/upstreamwatcher/upstreamwatcher.go

```diff
@@ -208,12 +207,11 @@ func (c *controller) validateIssuer(ctx context.Context, upstream *v1alpha1.Upst
 		var err error
 		discoveredProvider, err = oidc.NewProvider(ctx, upstream.Spec.Issuer)
```

Comment:
We should still log a (truncated?) version of this error string for debugging purposes.